### PR TITLE
LCD Test System 

### PR DESCRIPTION
**Pull request by NoahBreedy**
Base working test system for the TeenyAT

### CODE REVIEW

File: .gitignore
Comment (miniat-amos) [inline .gitignore:1]: None of these modifications make sense.  Remove.

File: docs/ISO.md
Comment (miniat-amos) [inline docs/ISO.md:1]: While we do need to introduce some of our TeenyAT documentation to the repo, it does not belong in a PR that introduces a new system.

**Review by miniat-amos — CHANGES_REQUESTED** [review]
There a only a handful of changes / questions that need to be addressed before I can merge.

**Comment by NoahBreedy** [comment]
The thing with the STR >= was not implemented by me. I didn't know
there was an error in regard to range checking.

**Comment by miniat-amos** [comment]
Merged... I restored the last good version of .gitignore, too.

**Pull request merged**